newscoop/src/Newscoop/NewscoopBundle/Session/Storage.php

```diff
@@ -24,10 +24,7 @@ class Storage extends PhpBridgeSessionStorage
     */
     public function __construct(array $options = array(), $preferencesService, $handler = null, MetadataBag $metaBag = null)
     {
-        $seconds = $preferencesService->SiteSessionLifeTime;
-
-        $options['cookie_lifetime'] = $seconds;
-        $options['gc_maxlifetime'] = $seconds;
+        $options['cookie_lifetime'] = $preferencesService->SiteSessionLifeTime;
```

Comment:
So the session lifetime still be used for the cookie lifetime? I don't think that's a good idea.